controllers/pwa.js

```diff
@@ -187,13 +187,28 @@ function renderPwaListPage(req, res) {
   const limit = parseInt(req.query.limit, 10) || LIST_PAGE_SIZE;
   const end = pageNumber * LIST_PAGE_SIZE;
   const contentOnly = false || req.query.contentOnly;
-  const search = false || req.query.search;
+
+  let mainPage = false;
```

Comment:
Maybe we could group those parameters in an object and create a method to extract the data from the request and return the object.